Softeq.XToolkit.HttpClient/Network/HttpRequestScheduler.cs

```diff
@@ -371,6 +371,18 @@ private async Task ExecuteAsync(CompleteHttpRequestScheduledTask task)
                     IsSuccessful = false
                 };
             }
+            catch (WebException ex)
+            {
+                //TODO PL: add handling on Android
+                if (ex.Message.ToLower().Contains("the internet connection appears to be offline"))
```

Comment:
Is it possible to move it to Default authorization library and handle exception there?